### ISSUE TITLE
Accessibility: Ctrl+Up/DownArrows does not work in terminal created in editor area

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#179979]
CC @meganrogge

Ctrl+Up/DownArrow (command history) does not work in terminal buffer created in the editor area.

**Comment by meganrogge** [comment]
With these steps, it works for me:

1. Run `Terminal: Create New Terminal in Editor Area`
2. Run 2 or 3 commands
3. `Shift+tab` into the accessible buffer
4. `Ctrl+ DownArrow/UpArrow` jumps to the command line

**Comment by meganrogge** [comment]
are you testing on the latest insider's and using a terminal with shell integration, such as pwsh?

**Comment by jooyoungseo** [comment]
So odd. Cannot repro anymore. Yes, I used latest insiders and PowerShell. Will reopen if I encounter this issue again.
